src/inference/src/ie_core.cpp

```diff
@@ -245,6 +282,7 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
     ov::SoPtr<ie::IExecutableNetworkInternal> compile_model_impl(const InferenceEngine::CNNNetwork& network,
                                                                  ov::InferencePlugin& plugin,
                                                                  const std::map<std::string, std::string>& parsedConfig,
+                                                                 const std::shared_ptr<ie::ICacheManager>& cacheManager,
                                                                  const ie::RemoteContext::Ptr& context,
                                                                  const std::string& blobID,
                                                                  const std::string& modelPath = std::string(),
```

Comment:
BTW, here we have a lot of parameters which are responsible for cache. Will it simplify the code if we create a dedicated structure `CacheContent` with `cacheManager`, `hash`, `blobID`, `modelPath ` `forceDisableCache` (and maybe other) fields?

Reply:
Done. Actually this structure will contain only 3 members: cacheManagerPtr, blobId (hash), modelPath. Last 2 members are optional